macro/Cargo.toml

```diff
@@ -27,6 +27,11 @@ syn = { workspace = true }
 quote = { workspace = true }
 subxt-codegen = { workspace = true, features = ["fetch-metadata"] }
 scale-typegen = { workspace = true }
+sc-executor = { workspace = true }
+sp-maybe-compressed-blob = { workspace = true }
+sp-state-machine = { workspace = true }
+sp-io = { workspace = true }
+sc-executor-common = { workspace = true }
```

Comment:
I wondfer how many dependencies we end up pulling in here; possibly worth putting behind a feature flag like "runtime_file_path" and then perhaps not enabling it by default (since I think it's probably a bit of an edge case that people would point straight to a runtime WASM blob).

Reply:
moved them to optional and hidden behind feature flag